CHANGELOG.md

```diff
@@ -21,11 +36,24 @@ The functions wrapping these APIs do not unwrap the single key JSON response, an
 ## Unreleased [2.x.x]
 
 ### Added
+
 - Enabled using both short and long IDs for playlist_change_details
+- Added a cache handler to `SpotifyClientCredentials`
 
 ### Changed
+
 - Add support for a list of scopes rather than just a comma separated string of scopes
 
+### Fixed
+
+* Fixed the bugs in `SpotifyOAuth.refresh_access_token` and `SpotifyPKCE.refresh_access_token` which raised the incorrect exception upon receiving an error response from the server. This addresses #645.
+
+* Fixed a bug in `RequestHandler.do_GET` in which the non-existent `state` attribute of  `SpotifyOauthError` is accessed. This bug occurs when the user clicks "cancel" in the permissions dialog that opens in the browser.
+
+* Cleaned up the documentation for `SpotifyClientCredentials.__init__`, `SpotifyOAuth.__init__`, and `SpotifyPKCE.__init__`.
+
```

Comment:
Since this PR goes straight into v3, v2 will never "see" these fixes. v2 features and fixes should still be merged into v2 (aka master)

Reply:
Actually, these changes that you referenced were merged into master in #655

Reply:
Oh my bad!